requests/models.py

```diff
@@ -420,8 +420,13 @@ def prepare_body(self, data, files, json=None):
         length = None
 
         if not data and json is not None:
+            # When urllib3 uses pyOpenSSL, it can only resume large uploads
```

Comment:
You're right; it's not relevant. I was in the mindset of urllib3 still, but in the context of requests, "the urllib3 API specifies a bytes-like body" is enough.